### DISCUSSION TITLE
Heroku API [Apr 02]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#1636] [Discord Threads]
> 🙋‍♂️ **paulc**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### Can you paste the curl request example from swagger.  
#  
> 💁 **o1lab**  
#### with token redacated ofcourse 🙂  
#  
> 💁 **o1lab**  
#### pls ensure you are using https  
#  
> 🙋‍♂️ **paulc**  
#### curl -X 'GET' 'https://wsbc-f9db.herokuapp.com/nc/project_1_cpnv/api/v1/Test/1' -H 'accept: application/json' -H 'xc-auth: ________'  
#  
> 🙋‍♂️ **paulc**  
#### it was http previously, returned an error.. with HTTPs it returns a 403 forbidden result, with the same previous message..  I've been playing outside of Heroku, spun up a VM with Vultr to play around with, and having similar issues - definitely not as smooth as installing on my VM at home, which I'm now trying to remember which way I did it, to try and replicate.. works fine in my local network but struggling to get it listening on the public IP.. The docs are good to a point, but lack a bit of detail for a couple of use cases (like non-docker, or public IP/proxying when in docker)..  feel like "I'll get there, but could use a nudge in the right direction" with a dose of "it shouldn't be this hard for a quick'n'dirty demo, I feel kinda dumb" haha  
#  
> 👨‍🚀 **wingkwong**  
#### are you using the auth token or api token for xc-auth?  
#  
> 🙋‍♂️ **paulc**  
#### I was using API token.. then realized the docs/example talked about the user auth token.. which worked great.. so maybe the API token isn't for what I thought it was for?  
#  
> 👨‍🚀 **wingkwong**  
#### API token s used to be integrated with 3rd party apps  
#  
> 🙋‍♂️ **paulc**  
#### Gotcha - reading up the docs more, it's a different header (-token instead of -auth).. I'll continue playing - thanks for the help!  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/958796241658064906)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
